Core/include/Acts/Utilities/Concepts.hpp

```diff
@@ -1,26 +0,0 @@
-// This file is part of the Acts project.
-//
-// Copyright (C) 2022 CERN for the benefit of the Acts project
-//
-// This Source Code Form is subject to the terms of the Mozilla Public
-// License, v. 2.0. If a copy of the MPL was not distributed with this
-// file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
-#pragma once
-
-#if defined(__cpp_concepts)
-
-#define ACTS_REQUIRES(x) requires(x)
-#define ACTS_CONCEPT(x) x
-#define ACTS_STATIC_CHECK_CONCEPT(check_concept, check_type) \
-  static_assert(check_concept<check_type>,                   \
-                #check_type " does not fulfill " #check_concept)
```

Comment:
Should we keep this as a convenience or just use the expanded `static_assert` everywhere?

Reply:
To me the direct use of `static_asset` looks cleaner